components/formats-bsd/src/loci/formats/in/OBFReader.java

```diff
@@ -494,7 +553,38 @@ else if (isFLIMLabel(label)) {
           stepLabels.add(list);
         }
         meta_data.seriesMetadata.put("StepLabels", stepLabels);
+
+        in.skipBytes(obsoleteMetadataLength);
+
+        if (numFlushPoints > 0) {
+            List<Long> flushPoints = new ArrayList<>((int)numFlushPoints);
+            for (int i = 0; i < numFlushPoints; ++i) {
+                flushPoints.add(in.readLong());
+            }
+            stack.flushPoints = flushPoints;
+        }
+
+        in.skipBytes(tagDictionaryLength);
+
+        if (numChunkPositions > 0) {
+          List<Long> logicalPositions = new ArrayList<>((int)numChunkPositions + 1);
+          List<Long> filePositions = new ArrayList<>((int)numChunkPositions + 1);
```

Comment:
Similar to ```flushPoints``` above; ```numChunkPositions``` is a ```long``` in the file, can this be reworked to prevent integer overflow?

Reply:
See above (also added an exception for this case).